lastgersync/lastgersync.go

```
@@ -69,16 +72,23 @@ func New(
 	}, nil
 }
 
-func (s *LastGERSync) Start(ctx context.Context) {
+// Start initiates the synchronization process.
+func (s *LastGERSync) Start(ctx context.Context) error {
+	if s.driver == nil {
+		return fmt.Errorf("driver is not initialized")
+	}
```

This should never happen, because from what I can see we are always invoking the `New` constructor function (which initializes the [driver](https://github.com/agglayer/aggkit/blob/99fae0611f85694e931c3eae0efa85d7101646c8/lastgersync/lastgersync.go#L64-L67)). WDYT?

```suggestion
```

This also means, that we wouldn't need to propagate an error from `Start` function.

---

You are right, I will remove this check